resources/cluster-users/templates/rbac-roles.yaml

```diff
@@ -215,6 +215,33 @@ aggregationRule:
       rbac.authorization.kyma-project.io/aggregate-to-kyma-kubeless-edit: "true"
 rules: []
 
+---
+################################################################################
+# kyma-snapshots = manage volume snapshots for backups.
+################################################################################
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-snapshots
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-snapshots: "true"
+  annotations:
+    helm.sh/hook-weight: "0"
+rules:
+- apiGroups:
+    - "snapshot.storage.k8s.io"
+  resources:
+    - "volumesnapshots"
+  verbs:
```

Comment:
There is a predefined list of "edit" verbs in values. You can use it here like this:
```
{{ toYaml .Values.clusterRoles.verbs.edit | indent 4 }}
```

Reply:
Applied your suggestion, thanks!